tensorflow/core/common_runtime/eager/execute.cc

```diff
@@ -999,7 +997,14 @@ Status EagerExecute(EagerOperation* op,
   bool op_is_local = op->EagerContext()->IsLocal(op->Device());
 
   if (op_is_local) {
-    return EagerLocalExecute(op, retvals, num_retvals);
+    std::unique_ptr<tensorflow::EagerOperation> out_op;
+    TF_RETURN_IF_ERROR(EagerOpRewriteRegistry::Global()->RunRewrite(
```

Comment:
This should probably happen at the top of the function to support remote eager operations as well.

Reply:
Done.